packages/ckeditor5-dev-env/lib/release-tools/tasks/update-package-versions.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+const { diffChars: diff } = require( 'diff' );
+const { execSync } = require( 'child_process' );
+const fs = require( 'fs' );
+const glob = require( 'glob' );
+
+/**
+ * Updates `@ckeditor/ckeditor5-*` and `ckeditor5` dependencies in `packages/*` and `release/*`
+ * directories to the latest version. Changes in `packages/*` will be committed as well.
```

Comment:
The description should be adapted, as there are no predefined `packages/*` and `release/*` directories in this script, but all paths to update are passed as an argument. The script does not know in advance which paths will be updated.